ts/components/session/settings/SessionSettingsHeader.tsx

```diff
@@ -6,19 +6,18 @@ import { SessionButton } from '../SessionButton';
 
 interface Props extends SettingsViewProps {
   showLinkDeviceButton: boolean | null;
```

Comment:
I think a comment about password lock being able to affect this and to why we need two different parameters here would be good.